packages/asc-web-common/api/settings/index.js

```diff
@@ -79,6 +79,49 @@ export function getLogoUrls() {
   });
 }
 
+export function getCustomSchema() {
```

Comment:
Rename to getCustomSchemas -> The method returns a list

Reply:
Rename to getCustomSchemasList.